### PR TITLE
bump build-number

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

We Just need to build the `win-64` platform in concourse since it is currently failing in prefect.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,23 +62,91 @@ requirements:
     - cytoolz >=0.7.3
     # cloudpickle is necessary to provide the 'processes' scheduler for dask
     - cloudpickle >=0.2.1
+    - _openmp_mutex  # [linux]
+    - tifffile >=2019.7.26
+    # scikit-image depends on dask-array
+    # which requires numpy, dask-core and toolz (cytoolz for speed)
+    - dask-core >=1.0.0,!=2.17.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:68]: dask-core and toolz are already mentioned above. It's a duplication

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,34 +7,51 @@ package:
...
-  # url: https://github.com/scikit-image/scikit-image/archive/v{{ version }}.tar.gz
-  # fn: scikit-image-{{ version }}.tar.gz
-  # sha256: 3b8414ccb9b524d0d507909e9a4d3843af6ca9919152199d380f2b21639abf17
+  patches:
+    - skip-broken-tests.patch
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py==27]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: Skip py<36, see https://github.com/scikit-image/scikit-image/blob/13a6ec15c0ea8c84abb7ab54b13b97fae67ae44a/setup.py#L233

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:68]: thanks I will remove it.  I just found an issue on py310 on Windows for tifffile.c header, which tries to use undeclared type 'ssize_t'.  I am testing a fix for it

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:16]: hmm, no.  we will need it for py 37 too, and 0.16.2 was supporting it in the past too.  btw testing passes fine on 3.7 for it

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**